aggregate/housing_security/households_rent_burden.py

```diff
@@ -0,0 +1,51 @@
+"""this chops up into six indicators they are """
+from typing import final
+import pandas as pd
+from aggregate.clean_aggregated import rename_col
+from utils.PUMA_helpers import clean_PUMAs, borough_name_mapper, get_all_boroughs, get_all_NYC_PUMAs
+from utils.dcp_population_excel_helpers import race_suffix_mapper, stat_suffix_mapper_ty
+from internal_review.set_internal_review_file import set_internal_review_files
+from aggregate.load_aggregated import load_clean_pop_data
+
+year_mapper = {"12": "0812", "19": "1519"}
+
+def households_rent_burden(geography: str, write_to_internal_review=False) -> pd.DataFrame:
+
+    name_mapper = {
+        "GRPI30": "households_rb",
+        "GRPI50": "households_erb",
+        "OHURt": "households_grapi", 
+    }
+
+    
+    ind_name_str = "|".join([k for k in name_mapper.keys()])
+    #print(ind_name_str)
+
+    clean_data = load_clean_pop_data(ind_name_str)
+
+    if geography == "citywide":
```

Comment:
this if/else block can be abstracted into it's own function I think. fine for this PR but for future a good refactor